assignments/objects.js

```diff
@@ -19,26 +19,53 @@ const example = {
 
 // Write your intern objects here:
 
+function populate(ID, email, name, sex) {
+  const employee = {
+    "id": ID,
+    "name": name,
+    "email": email,
+    "gender": sex,
+    "speak": function(){
+      return "Hello, my name is " + name;
+    },
+    "multiplyNums": function(a, b) {
+      return a * b;
+    }
+  }
+  return employee;
+}
+
+const mitzi = populate(1,"mmelloy0@psu.edu","Mitzi","F");
+
+const kennan = populate(2,"kdiben1@tinypic.com","Kennan","M");
+
+const keven = populate(3,"kmummery2@wikimedia.org","Keven","M");
+
+const gannie = populate(4,"gmartinson3@illinois.edu","Gannie","M");
+
+const antonietta = populate(5,"adaine5@samsung.com","Antonietta","F");
```

Comment:
This is rad, if you didn't know this is called a constructor function.  We'll get into those tomorrow I think.